functions.sh

```diff
@@ -351,6 +351,26 @@ build_env() {
                     mv build/"${pieces[0]}"/chroot "${TARGET}/${pieces[0]}"
                 done
             popd
+
+            chroot_desktop=(
+                "ubuntu-bionic-arm64,xfce4,rpi-3-b"
+                "ubuntu-bionic-armhf,xfce4,rpi-3-b"
+            )
+
+            pushd "${TARGET}"/pieman
+                for chroot in "${chroot_desktop[@]}"; do
+                    IFS=',' read -r -a pieces <<< "${chroot}"
+                    env CREATE_ONLY_CHROOT=true \
+                        OS="${pieces[0]}" \
+                        PROJECT_NAME="${pieces[0]}-${pieces[1]}" \
+                        DEVICE="${pieces[2]}" \
+                        PYTHON="${TARGET}"/pieman-env/bin/python \
+                        XFCE4=true \
```

Comment:
@leocoan I guess where would be a problem when we add more desktop environments with `XFCE4=true`. Would be great if you can suggest how we can update this code if we add `GNOME=true` for example.

Reply:
It's unnecessary. We will copy CREATE_ONLY_CHROOT block and set XFCE4=false and CNOME(and other)=true